specification/hugr.md

```diff
@@ -1037,8 +1033,8 @@ Tuples and Sums are ``CopyableType`` (respectively, ``EqType``) if all their com
 
 The type of `Function` includes a set of extensions (that is, [Extensions](#extension-implementation)) which are required to execute the graph. Every node in the HUGR is annotated with the set of extensions required to produce its inputs, and the set of extensions required to execute the node; the union of these two must match the set of extensions on each successor node.
 
-Keeping track of the extension requirements like this allows extension designers and backends
-(like tierkreis) to control how/where a module is run.
+Keeping track of the extension requirements like this allows extension designers
+and backends to control how/where a module is run.
```

Comment:
Is backends really the right word here? Typically I think this kind of check would be done by the *frontend* in order to avoid making an impossible request to a backend (i.e. "run this" when the backend can't)...

Reply:
went with "third-party tooling"